### ISSUE TITLE
Training script can't read dataset?

### ISSUE THREAD

**Issue by SoulflareRC** [KichangKim/DeepDanbooru#85]
Hi I was trying to train the model using the commandline tool, but I'm not sure if the data is actually read and used to train the model. The training completed instantly for me and I don't think it has actually read the data, as the image below shows
![image](https://user-images.githubusercontent.com/107384280/216802456-7d41ee90-c633-4288-9af4-de7de822c89d.png)
My dataset folder's directory tree looks like this 
![image](https://user-images.githubusercontent.com/107384280/216802577-5570b2cb-62fe-4891-83e8-87ce9f5d39b4.png)
and my-dataset.sqlite file is in the same folder as images folder is in. 
Here is my project.json
`{
    "image_width": 299,
    "image_height": 299,
    "database_path": "D:\\pycharmWorkspace\\DeepDanbooru\\test\\MyDataset\\my-dataset.sqlite",
    "minimum_tag_count": 0,
    "model": "resnet_custom_v2",
    "minibatch_size": 32,
    "epoch_count": 10,
    "export_model_per_epoch": 10,
    "checkpoint_frequency_mb": 200,
    "console_logging_frequency_mb": 10,
    "loss": "binary_crossentropy",
    "optimizer": "adam",
    "learning_rate": 0.001,
    "rotation_range": [
        0.0,
        360.0
    ],
    "scale_range": [
        0.9,
        1.1
    ],
    "shift_range": [
        -0.1,
        0.1
    ],
    "mixed_precision": false
}`

What am I missing here? Could you help with this issue? 
Thank you.

**Comment by KichangKim** [comment]
Make sure that your image filenames have all lower-case.

**Comment by SoulflareRC** [comment]
Hi thank you for replying! My filenames are all md5 so I'm pretty sure filenames are not an issue. I was suspecting the issue was from the image size. Do all the images have to be 299x299(or generally the same size)? Can I change this parameter in the json file?

**Comment by KichangKim** [comment]
You don't need to resize your image files. It can be any size and DeepDanbooru resize it automatically.
